### ISSUE TITLE
Support for other OSes

### ISSUE THREAD

**Issue by leogdion** [hiddevdploeg/Billboard#13]
Primarily macOS but watchOS would be 🔥

**Comment by dhaydl** [comment]
I would love to try implementing this. is this possible?

**Comment by leogdion** [comment]
I don't think watchOS has a hookup but macOS shouldn't be too difficult to design

**Comment by hiddevdploeg** [comment]
I'm not sure it makes sense to have ads on your wrist as they often would take at least a few seconds and lifting your arm is tough as is.

macOS is definitely welcome :)

**Comment by dhaydl** [comment]
sure, but what about ad cards for a List for example?

**Comment by leogdion** [comment]
yeah agreed @hiddevdploeg until Apple comes out with that $299 Arm Sling
macOS is lower hanging fruit in many ways.
